setup.py

```diff
@@ -59,6 +59,7 @@
             "null.qubit = pennylane.devices.null_qubit:NullQubit",
             "default.qutrit = pennylane.devices.default_qutrit:DefaultQutrit",
             "default.clifford = pennylane.devices.default_clifford:DefaultClifford",
+            "default.qutrit.mixed = pennylane.devices.default_qutrit_mixed:DefaultQutritMixed",
```

Comment:
I added the device to the setup.py file as it should be ready to use and it makes it consistent with other devices.